Tests/DatadogBenchmarkTests/DataStorage/LoggingStorageBenchmarkTests.swift

```diff
@@ -19,10 +19,12 @@ class LoggingStorageBenchmarkTests: XCTestCase {
         try super.setUpWithError()
         self.directory = try Directory(withSubdirectoryPath: "logging-benchmark")
 
-        let storage = LoggingFeature.createStorage(
-            directories: FeatureDirectories(
+        let storage = FeatureStorage(
+            featureName: "logging",
+            dataFormat: DataFormat(prefix: "[", suffix: "]", separator: ","),
+            directories: .init(
                 deprecated: [],
-                unauthorized: obtainUniqueTemporaryDirectory(),
+                unauthorized: directory,
                 authorized: directory
             ),
```

Comment:
It now becomes pointless to have separate storage benchmarks for Logs, Traces and RUM as all use the same logic under the hood. To make CI pass, I'm just updating it for now, but we can re-consider Benchmarks module as part of V2 migration. It will be tedious in maintenance.